lab-dustin/src/component/note-list-container/index.js

```diff
@@ -0,0 +1,25 @@
+import './_note-list-container.scss';
+import React from 'react';
+import uuid from 'uuid/v1';
+import NoteCreateForm from '../note-create-form-container';
+import NoteItem from '../note-container';
```

Comment:
Calling note-container NoteItem here is just going to cause confusion. Always name it the exact same as the file itself.